test/lolex-test.js

```diff
@@ -1622,10 +1622,10 @@ describe("lolex", function () {
             assert(Date.prototype.isPrototypeOf(date));
         });
 
```

Comment:
Please don't change the existing tests to test for the string. Revert the changes to the test file and change them to create a new instance by adding `new `. As in `new this.clock.Date()`

Then add a new test for the behaviour you are testing for. Which is to test that when _not_ called with new, it returns a string. You don't need to test for all the cases; I'd say it suffices to test for a single case to know the logic is sound.

Reply:
Hi, thanks. Actually tests for each set of parameters for calling Date as a function were created before, I just fixed result. You can take a look there are two tests for each set of parameters in master:

![Screenshot from 2019-08-22 16-14-52](https://user-images.githubusercontent.com/891497/63546546-11bc5700-c4f8-11e9-97b2-90b5e2c45715.png)

Should I remove those tests?

Reply:
Yeah, I think the old tests were just _wrong_, no?